src/native/containers/dn-simdhash-specializations.h

```diff
@@ -61,7 +61,7 @@ typedef struct dn_simdhash_str_key dn_simdhash_str_key;
 
 
 typedef struct dn_ptrpair_t {
-    void *first, *second;
+	void *first, *second;
```

Comment:
It looks like a couple of 4 spaces got accidentally converted to tabs at several places.

Reply:
That's on purpose, these files are supposed to be tab indented. For some reason vscode randomly converts tabs into spaces when editing so I have to manually switch it back.